config.json

```diff
@@ -341,7 +341,10 @@
       "unlocked_by": null,
       "difficulty": 1,
       "topics": [
-
+        "mathematics",
+        "floating-point-numbers",
```

Comment:
Could you please convert it to `snake_case` for consistency with https://github.com/exercism/problem-specifications/blob/master/TOPICS.txt